### PR TITLE
Seed the RNG before signing the package.

### PR DESCRIPTION
**Pull request by mpcomplete**

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: lib/src/commands/build.dart [hunk 1]
```diff
@@ -182,6 +182,8 @@ class BuildCommand extends FlutterCommand {
         archive.addFile(file);
     }
 
+    await CipherParameters.get().seedRandom();
```
Comment (devoncarew) [inline lib/src/commands/build.dart]: I think `seedRandom` will throw on windows; perhaps put an OS check around this?

Comment (mpcomplete) [inline lib/src/commands/build.dart]: Right. How can I check the OS?

Comment (mpcomplete) [inline lib/src/commands/build.dart]: Actually, I'll change `seedRandom` to not throw.

Comment (devoncarew) [inline lib/src/commands/build.dart]: Cool. There's a Platform class in dart:io with some properties like isWindows.

**Pull request merged**